daemon/src/model/cfd.rs

```diff
@@ -349,6 +350,13 @@ pub struct SettlementProposal {
     pub maker: Amount,
 }
 
+#[derive(Debug, Clone)]
+#[allow(dead_code)] // Variants used by different binaries
+pub enum SettlementProposals {
+    Incoming(HashMap<OrderId, SettlementProposal>),
+    Outgoing(HashMap<OrderId, SettlementProposal>),
+}
```

Comment:
Does this enum actually bring any advantages? I think we could split this into a single hashmap per binary no?

Reply:
I think a better data structure would be:

```rust
HashMap<OrderId, (SettlementProposal, IncomingOrOutgoing)>
```

Reply:
The difference would be that it means that we *could* have both incoming and outgoing settlement proposals in the taker and maker. the way as it is currently you can only setup the channel with either, and it can catch some programmer errors (e.g. as it did when I was developing :D ) 

Of course, if we even decide to handle both incoming and outgoing proposals the code will change.

Reply:
Hmm, I think eventually we want both parties to propose collaborative settlement.

Reply:
ok, I'll look into simplifying the data structure as a separate PR.

Reply:
https://github.com/comit-network/hermes/pull/215